srcpkgs/duplicati-beta-bin/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'duplicati-beta-bin'
+pkgname=duplicati-beta-bin
+version=2.0.4.5
+revision=1
+noarch=yes
+__rel_date="2018-11-28"
+__reldate="20181128"
+build_style=fetch
+hostmakedepends="rpmextract"
+depends="gtk-sharp2 mono"
+short_desc="Backup software to store encrypted backups online"
+maintainer="mobinmob <mobinmob@disroot.org>"
+#Duplicati is licensed under LGPL-2.1-or-later, all other licenses are for bundled third-party components.
+license="LGPL-2.1-or-later BSD-3-Clause GPL-2.0-or-later MIT Apache-2.0 LGPL-3.0-only MicrosoftSharePointClientComponentsEULA MS-PL Public Domain"
+homepage="https://www.duplicati.com/"
+distfiles="https://github.com/duplicati/duplicati/releases/download/v${version}-${version}_beta_${__rel_date}/duplicati-${version}-${version}_beta_${__reldate}.noarch.rpm"
+checksum=97a2a0445fc869c0e36a3815905a7325168172b6ac4742c66bcf85c6ceb77e2e
+
+
+do_install() {
+	cd ${DESTDIR} && rpmextract "$wrksrc"/duplicati-${version}-${version}_beta_${__reldate}.noarch.rpm
+	vmkdir usr/share/licenses/duplicati
+	mv usr/lib/duplicati/licenses/* usr/share/licenses/duplicati
```

Comment:
```
for l in usr/lib/duplicate/licenses/*; do
    vlicense $l
done
```

Reply:
Doesn 't work. The licenses are in subdirectories and there are multiple files with the same name across subdirs.